inputsandstrings/js-inputs-and-strings_change-box-appearance/js/index.js

```diff
@@ -16,3 +16,27 @@ For further information check MDN:
 - about CSS declarations with "style": https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/style
 
 */
+
+const inputColor = document.querySelector('[data-js="input-color"]');
+const inputRadius = document.querySelector('[data-js="input-radius"]');
+const inputRotation = document.querySelector('[data-js="input-rotation"]');
+const box = document.querySelector('[data-js="box"]');
+
+inputColor.addEventListener("input", function () {
+  const inputColorValue = this.value;
```

Comment:
you don't need to say 'function', you can replace it with => e.g. 
_addEventListener("input", () => {_